neuro_san/internals/run_context/langchain/core/langchain_run_context.py

```diff
@@ -27,12 +27,12 @@
 
 from pydantic_core import ValidationError
 
-from langchain.agents import Agent
-from langchain.agents import AgentExecutor
-from langchain.agents.tool_calling_agent.base import create_tool_calling_agent
-from langchain.base_language import BaseLanguageModel
-from langchain.callbacks.tracers.logging import LoggingCallbackHandler
+from langchain_classic.agents import Agent
+from langchain_classic.agents import AgentExecutor
+from langchain_classic.agents.tool_calling_agent.base import create_tool_calling_agent
+from langchain_classic.callbacks.tracers.logging import LoggingCallbackHandler
```

Comment:
Interesting to see some necessary imports move to what's now called "langchain_classic".
Comments in langchain_core point to using langgraph as the basis library for agents.

For this first blush we will use langchain_classic.